src/JustSaying/AwsTools/MessageHandling/SqsQueueByUrl.cs

```diff
@@ -1,33 +0,0 @@
-using System;
-using System.Linq;
-using System.Threading.Tasks;
-using Amazon;
-using Amazon.SQS;
-using Amazon.SQS.Model;
-using Microsoft.Extensions.Logging;
-
-namespace JustSaying.AwsTools.MessageHandling
-{
-    public class SqsQueueByUrl : SqsQueueBase
```

Comment:
It feels good to be rid of this - was it ever used anywhere?

Reply:
Yeah, unused, maybe should be a separate PR 🤷